pkg/proxy/proxy.go

```diff
@@ -242,7 +241,7 @@ func (p *Proxy) Get(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		logger.Error("Could not get load test info with error:", zap.Error(err))
 
-		if k8sErr, ok := err.(k8sAPIErrors.APIStatus); ok && k8sErr.Status().Code == http.StatusNotFound {
+		if k8sAPIErrors.IsNotFound(err) {
```

Comment:
Seems like this is unrelated to the protoc makefile changes can you split this PR please.

Reply:
It is already in another dedicated commit.